src/sqe.rs

```diff
@@ -155,6 +265,26 @@ impl<'a> SubmissionQueueEvent<'a> {
         uring_sys::io_uring_prep_fsync(self.sqe, fd, flags.bits() as _);
     }
 
+    /// Prepare a timeout event.
+    /// ```
+    /// # use iou::IoUring;
+    /// # use iou::sys;
```

Comment:
A few doc tests won't compile anymore because they import `iou::sys`, thanks to #20. You can fix this by using `uring_sys` instead of `iou::sys`.